js/src/lib/interfaces/Libraries.ts

```diff
@@ -148,6 +149,28 @@ const keras = (model: ModelData) =>
 model = from_pretrained_keras("${model.id}")
 `;
 
+const paddlenlp = (model: ModelData) => {
+	const info = model.transformersInfo;
+	if (!info) {
+		return `# ⚠️ Type of model unknown`;
+	}
+	if (info.processor) {
+		return [
+			`from paddlenlp.transformers import AutoTokenizer, ${info.auto_model}`,
+			"",
+			`tokenizer = AutoTokenizer.from_pretrained("${model.id}"${model.private ? ", use_auth_token=True" : ""}, from_hf_hub=True)`,
+			"",
+			`model = ${info.auto_model}.from_pretrained("${model.id}"${model.private ? ", use_auth_token=True" : ""}, from_hf_hub=True)`,
+		].join("\n");
+	} else {
+		return [
+			`from paddlenlp.transformers import ${info.auto_model}`,
+			"",
+			`model = ${info.auto_model}.from_pretrained("${model.id}"${model.private ? ", use_auth_token=True" : ""}, from_hf_hub=True)`,
+		].join("\n");
+	}
+};
+
```

Comment:
To simplify this PR as the information from the PaddleNLP is not in the `transformersInfo` as the config/library is different, would it be possible to do a very high-level snippet and afterward we could increase the coverage? First, we need this PR merged to start working on the internal side of things to add more info to the `ModelData`. Something along these lines


```suggestion
const paddlenlp = (model: ModelData) => {
	return [
	  `from paddlenlp.transformers import AutoModel`,
	  "",
	  `model = AutoModel.from_pretrained("${model.id}"${model.private ? ", use_auth_token=True" : ""}, from_hf_hub=True)`,
        ].join("\n");
};

```

Reply:
Thanks for the suggestion. I basically took your suggestion. Only added a `AutoTokenizer`